src/main/scala/com/datastax/gatling/plugin/model/DseCqlAttributesBuilder.scala

```diff
@@ -123,8 +150,8 @@ case class DseCqlAttributesBuilder(attr: DseCqlAttributes) {
     * @return
     */
   @deprecated("Replaced by withSerialConsistencyLevel")
-  def serialConsistencyLevel(level: ConsistencyLevel) = withSerialConsistencyLevel(level)
-
+  def serialConsistencyLevel(level: ConsistencyLevel):DseCqlAttributesBuilder[T, B] =
```

Comment:
Given the plugin already has breaking changes, this method can be removed, as it was already deprecated